rosidl_typesupport_opendds_c/resource/srv__type_support_c.cpp.em

```diff
@@ -108,53 +107,22 @@ static void * create_requester__@(service.namespaced_type.name)(
   void ** untyped_writer,
   void * (*allocator)(size_t))
 {
-  return @('::'.join(service.namespaced_type.namespaces))::typesupport_opendds_cpp::create_requester__@(service.namespaced_type.name)(
-    untyped_participant,
-    request_topic_str,
-    response_topic_str,
-    untyped_datareader_qos,
-    untyped_datawriter_qos,
-    untyped_reader,
-    untyped_writer,
-    allocator);
+  return NULL;
 }
 static const char * destroy_requester__@(service.namespaced_type.name)(
   void * untyped_requester,
   void (* deallocator)(void *))
 {
-  return @('::'.join(service.namespaced_type.namespaces))::typesupport_opendds_cpp::destroy_requester__@(service.namespaced_type.name)(
-    untyped_requester, deallocator);
+@# TODO: Implement
```

Comment:
Is "implement" as much as you know about what needs to be done here and for the other `TODO`s?